src/Translator/Translator.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Yiisoft\I18n\Translator;
+
+use Psr\EventDispatcher\EventDispatcherInterface;
+use Yiisoft\I18n\Event\MissingTranslationEvent;
+use Yiisoft\I18n\TranslatorInterface;
+
+/**
+ * TODO may be it must be taken out from yiisoft/i18n?
```

Comment:
No, it belongs to the package. Let's have it here.

Reply:
ok